src/pybind/mgr/dashboard/tests/test_grafana.py

```diff
@@ -38,13 +37,17 @@ def test_validation(self):
         self._get('/api/grafana/validation/foo')
         self.assertStatus(500)
 
-    def test_dashboards(self):
+    def test_dashboards_1(self):
```

Comment:
I would prefer a better test name (same for the tests beneath in this file)

```suggestion
    def test_dashboards_unavailable_no_url(self):
```

Reply:
Done